src/components/Router.js

```diff
@@ -69,6 +71,13 @@ export default class Router extends Component {
     this.unlisten = Nav.history.listen(
       ({ pathname, search }) => this.setState({ pathname, search })
     )
+
+    if (isFirecloud() && new URLSearchParams(window.location.search).has('fcredir')) {
+      notify('', 'Welcome to Terra, the next version of FireCloud. All of your workspaces, data, notebooks, ' +
```

Comment:
Even though an empty string seems to work, this should probably specify the `'info'` type explicitly.